tests/tests/tier0/bluechi-is-online-agent/main.fmf

```diff
@@ -0,0 +1,2 @@
+summary: Test if bluechi agent is online
```

Comment:
Please make it clear that the test uses `bluechi-is-online` tool with `agent` parameter, for example:

```
summary: Test if bluechi-agent is online using `bluechi-is-online agent` command line tool
```